jawa/methods.py

```diff
@@ -60,10 +60,22 @@ def code(self):
         """
         A shortcut for::
 
-            method.attributes.find_one(name='Code')
+            >> method.attributes.find_one(name='Code')
         """
         return self.attributes.find_one(name='Code')
 
+    @property
+    def bootstrap(self):
```

Comment:
As per [JVMS §4.7.23](https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.7.23), `BootstrapMethods` is in the ClassFile's attributes, not the attributes for an individual method; this property should be in ClassFile.